docs/source/chainermn/installation/guide.rst

```diff
@@ -94,7 +94,7 @@ CuPy
 ~~~~
 
 Chainer and ChainerMN rely on CuPy to use GPUs. 
```

Comment:
The previous one is correct: although it depends on the Sphinx configuration, but we need "__"  (double underline) suffix to links.

Reply:
I see, thanks.